examples/controls/image.rs

```diff
@@ -8,6 +8,9 @@ element {
 }
 "#;
 
+#[cfg(target_arch = "wasm32")]
+compile_error!("This example uses image loading that does not work on the web");
```

Comment:
I looked into this a bit before when trying to make a web app with images. The part that doesn't work on web is actually spawning a thread, the image loading itself works. However, I also discovered that the image only works when _not_ set to repeat in x and y.

Oh and loading an image from the web doesn't work either, but I think it could with an async version.